src/k8s/pkg/k8sd/pki/worker.go

```diff
@@ -49,3 +49,7 @@ func (c *WorkerNodePKI) CompleteCertificates() error {
 	}
 	return nil
 }
+
+func (c *WorkerNodePKI) IsKubeletPresent() bool {
```

Comment:
I get that this operates on a WorkerNodePKI object but could we name it something like "AreKubeletCertsSet" so the call would look like "certificates.AreKubeletCertsSet()"? I think it'd be more immediately clear what the function is checking for.